src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html

```diff
@@ -1,5 +1,7 @@
 <div>
-    A personal access token with project admin permission. This is used for things that require elevated permissions
-    (such as creating webhooks).
-    This token is not visible to users, and is not used to access projects and/or repositories.
+    <p>Adding a <a href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Bitbucket
+        personal access token</a> allows users to automatically set up build triggers when creating a Jenkins project.
+        Importantly, for this to work the token must have project admin permissions.</p>
+
+    <p>Users will only be able to use the token to set up build triggers.</p>
```

Comment:
I think this text is wrong here. If I come in and I wonder "what does this 'token' field need?" This text is not helping me. This text is for the **field** so I think we should be far more specific 
```html
<p>This is the value of the token that was displayed when the token was created in Bitbucket Server</p>
```
or something similar.

Reply:
I've addressed this by kind of mixing them together- preserving the hyperlink and inserting that into your sentence, and keeping the following text.

Reply:
Hi @dkjellin.  You're right and it should be more of an instruction. The only thing I'm not sure about is whether we need to say that the token value is displayed when the token is created. 

In the docs we recommend using one token per integration. I think we should mention this here. This means that telling users when the token value is displayed isn't needed. I also think it's worth mentioning the benefits again in case they don't read the first help text. 

I think we should change this to: 

> Here you should enter a [personal access token] https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens) with project admin permissions that you created in Bitbucket. This will allow users to automatically set up build triggers when creating a Jenkins project. 
> 
> We recommend creating a new token to use in this plugin. This way, if this token needs to be removed no other plugins will be affected. 
>
>Users will only be able to use the token you add to set up build triggers.